grid-proxy/pkg/types/nodes.go

```diff
@@ -140,7 +140,7 @@ type NodeFilter struct {
 	Rented            *bool    `schema:"rented,omitempty"`
 	RentedBy          *uint64  `schema:"rented_by,omitempty"`
 	AvailableFor      *uint64  `schema:"available_for,omitempty"`
-	NodeID            *uint64  `schema:"node_id,omitempty"`
+	NodeID            []uint64 `schema:"node_id,omitempty"`
```

Comment:
this should  be `NodeIDs instead to reflect it is a list 
 also consider changing it in `schema:"node_ids,omitempty"``
 and this for all relevant cases in other files